plugins/sysinfo/sysinfo.cpp

```diff
@@ -338,7 +340,10 @@ getWmiInfo (int mode)
 				hr = pclsObj->Get (L"Name", 0, &vtProp, 0, 0);
 				break;
 		}
-		WideCharToMultiByte (CP_ACP, 0, vtProp.bstrVal, -1, buffer, SysStringLen (vtProp.bstrVal)+1, NULL, NULL);
+		buffer_iter += WideCharToMultiByte(CP_ACP, 0, vtProp.bstrVal, -1, buffer_iter, SysStringLen(vtProp.bstrVal) + 1, NULL, NULL) - 1;
+		*(buffer_iter++) = ',';
```

Comment:
I haven't ran this or anything, but isn't adding "," at the end everytime wrong? It should be added at the start of a second loop if you have more than one?